### PR TITLE
Fix readme

### PR DESCRIPTION
**Pull request by Cyb3r-Jak3**
## Bug Fixes

Fix issue with hyperlink for the downloads badge

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -3,24 +3,24 @@ Haralyzer
 =========
 
 .. image:: https://badge.fury.io/py/haralyzer.svg
-    :target: http://badge.fury.io/py/haralyzer
+   :target: http://badge.fury.io/py/haralyzer
```
Comment (mrname) [inline README.rst:6]: Maybe go ahead and switch protocol to HTTPS while we are updating stuff?

**Comment by mrname** [comment]
Looks like Ubuntu 16 checks are stuck... are they deprecated or is GH just having troubles?

File: README.rst (hunk 1 above)
Comment (Cyb3r-Jak3) [reply README.rst]: Good calll

**Comment by Cyb3r-Jak3** [comment]
No idea what is going on with the ubuntu checks. Given 16.04 LTS is EOL as of April 2021 I'm going to drop all ubuntu 16 checks

**Review by mrname — APPROVED** [review]
Nice work, looks good! Not sure what happened with the failing check but it's up to you as to whether you want to spend time on that or not.

**Comment by Cyb3r-Jak3** [comment]
Seems to be a memory error on that test. I'm going to ignore for now but it keeps popping up I can take a look.

**Pull request merged**